audit_exceptions/mismatched_binary_allowlist.json

```diff
@@ -1,7 +1,7 @@
 {
   "aws-sam-cli": "libexec/lib/python*/site-packages/samcli/local/rapid/aws-lambda-rie-*",
   "balena-cli": "**/*",
-  "faust": "share/faust/android/app/lib/libsndfile/lib/*/libsndfile.so",
-  "lima": "share/lima/lima-guestagent.Linux-*",
+  "faust": "share/faust/android/app/lib/libsndfile/*/libsndfile.so",
+  "lima": "share/lima/lima-guestagent.Linux-a*",
```

Comment:
At least seems to work as intended when using incomplete glob, e.g. https://github.com/Homebrew/homebrew-core/actions/runs/8413974251/job/23037083115?pr=167024
```
  faust
    * Binaries built for a non-native architecture were installed into faust's prefix.
      The offending files are:
        /home/linuxbrew/.linuxbrew/Cellar/faust/2.70.3/share/faust/android/app/lib/libsndfile/lib/arm64-v8a/libsndfile.so	(arm64)
        /home/linuxbrew/.linuxbrew/Cellar/faust/2.70.3/share/faust/android/app/lib/libsndfile/lib/armeabi-v7a/libsndfile.so	(arm)
```
```
  lima
    * Binaries built for a non-native architecture were installed into lima's prefix.
      The offending files are:
        /home/linuxbrew/.linuxbrew/Cellar/lima/0.21.0/share/lima/lima-guestagent.Linux-riscv64	(dunno)
  Error: 1 problem in 1 formula detected.
```